src/abstract_xml_object.js

```diff
@@ -46,17 +47,30 @@ AbstractXMLObject.prototype.createElementInput = function (inputID, startingValu
 			|| this.objectType.attribute || this.objectType.cdata || this.objectType.comment){
 		input = document.createElement('textarea');
 		input.id = inputID;
+		input.rows = 1; // added start size of 1 row
+		input.style.height = '18px'; // Predefined height for 1 row. Will be expanded using jQuery.autosize
```

Comment:
Would it work for you to override the height setting in the `.attribute_container > textarea` rule instead?  Just a bit iffy on hardcoding a height in here unless necessary